src/main/java/com/example/onlinebookstore/service/book/BookServiceImpl.java

```diff
@@ -17,6 +19,7 @@ public class BookServiceImpl implements BookService {
 
     private final BookRepository bookRepository;
     private final BookMapper bookMapper;
+    private final CategoryRepository categoryRepository;
 
     @Override
     public BookDto save(CreateBookRequestDto requestDto) {
```

Comment:
Can you save the book with categories? Let’s fix it ;) (check also update)

Reply:
I do not quite understand

Reply:
answered in chat